src/gui/dialogs/multiplayer/lobby.cpp

```diff
@@ -693,12 +694,16 @@ void mp_lobby::update_playerlist()
 		tree_group_field["use_markup"] = "true";
 		tree_group_item["name"] = tree_group_field;
 
-		tree_view_node& player = target_list->tree->add_child("player", tree_group_item);
+		//tree_view_node& player = target_list->tree->add_child("player", tree_group_item);
 
-		connect_signal_mouse_left_double_click(find_widget<toggle_panel>(&player, "tree_view_node_label", false),
-			std::bind(&mp_lobby::user_dialog_callback, this, &user));
+		//connect_signal_mouse_left_double_click(find_widget<toggle_panel>(&player, "tree_view_node_label", false),
```

Comment:
Is this missing in the new implementation? I don't see anything that connects signals.

Reply:
Yeah, it's currently incomplete.